# Define the goal
string tool_name      # 'name' in suction_control.yaml
bool turn_suction_on  #  True: on, False: off
bool eject            #  True: expel air, False: do nothing
---
# Define the result
bool success
---
# Define a feedback message
uint32 suctioned      # digital_input of ur_control
